### DISCUSSION TITLE
Error while trying to use geoai.train_timm_segmentation_model

### DISCUSSION THREAD

**Discussion by Sumana18** [opengeos/geoai#331] [Q&A]
I updated the geoai package and tried to use the `geoai.train_timm_segmentation_model`, but I am getting an error with the `verbose` keyword. 
`data_folder = "labels_6_combined"
geoai.train_timm_segmentation_model(
    images_dir=f"{data_folder}/images",
    labels_dir=f"{data_folder}/labels",
    output_dir=f"{data_folder}/unet_resnet50",
    encoder_name="resnet50",
    architecture="unet",
    encoder_weights="imagenet",
    num_channels=8,  # PlanetScope has 8 bands
    num_classes=2,  # background and ALM polygons
    batch_size=8,
    num_epochs=100,
    learning_rate=0.00001,
    val_split=0.3,
    seed = 42,
    monitor_metric='val_loss',
    mode='min',
    patience=10,
    save_top_k=1,
    verbose=True,
)`

<img width="860" height="863" alt="image" src="https://github.com/user-attachments/assets/47c35d73-cd47-4e42-b0f6-ea29b67b269b" />

**Comment by giswqs** [comment]
Try out this notebook to see if you get the same errors: 
https://opengeoai.org/examples/train_timm_segmentation

**Reply by Sumana18** [reply]
Yes, I am still getting the same error.
<img width="1262" height="636" alt="image" src="https://github.com/user-attachments/assets/feca12d3-921b-4033-946e-38cd77b8a2cb" />
<img width="1269" height="612" alt="image" src="https://github.com/user-attachments/assets/acef49b0-56e2-41cf-8e3d-d30b69e183f5" />

**Comment by Sumana18** [comment]
Getting the same error while trying to run the code in Google Colab. 
<img width="1697" height="1285" alt="image" src="https://github.com/user-attachments/assets/de7cbe0f-e079-42db-ba64-c6d0edce9e52" />
